var/spack/repos/builtin/packages/visit/package.py

```diff
@@ -95,6 +95,7 @@ class Visit(CMakePackage):
     patch("cmake-findvtkh-3.3.patch", when="@3.3.0:3.3.2+vtkm")
     patch("cmake-findjpeg.patch", when="@3.1.0:3.2.2")
     patch("cmake-findjpeg-3.3.patch", when="@3.3.0")
+    patch("0001-fix-missing-header-includes-for-QSurfaceFormat.patch", when="+gui")
```

Comment:
Does this work with all the supported versions of `visit`?

Reply:
I have only tested with 3.3.3, so I've changed it to `@3.3.3+gui`. I might be able to test a few other versions, but it takes too long to build to exhaustively test the available versions.